### ISSUE TITLE
There is no options to accept new values into the combobox

### ISSUE THREAD

**Issue by mfissehaye** [danielfarrell/bootstrap-combobox#155]
The combobox can only hold values within the select element. It would be better if there was an option to let a user enter a new value (that doesn't exist in the options list)

**Comment by asecondwill** [comment]
+1, it's not a combobox without ability to add new item.

**Comment by TroyWolf** [comment]
+1 Definition of "combobox" is a dropdown list of options with the ability to enter your own value not in the list. That's what I need and why I found this with my search. I looked through the nice examples and thought, this is missing the "combo" in combobox!

**Comment by tjcrowder** [comment]
Indeed, this is not a combobox. More on comboboxes here: https://en.wikipedia.org/wiki/Combo_box
